packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController_Test.h

```diff
@@ -4,26 +4,18 @@
 
 #import "GoogleMapMarkerController.h"
 
+/// Methods exposed for unit testing.
 @interface FLTGoogleMapMarkerController (Test)
 
-/// Extracts an icon image from the iconData array.
```

Comment:
Both of these were dead code; they were exposing helpers that no longer exist in this class, and are instead stand-alone functions in a util file that are tested directly. I just missed this cleanup when converting them.